packages/commonwealth/client/scripts/views/components/Comments/CreateComment.tsx

```diff
@@ -171,7 +168,7 @@ export const CreateComment = ({
           )}
           <CWButton
             buttonWidth="wide"
-            disabled={disabled}
+            disabled={disabled && !isAdmin}
```

Comment:
Can you add this `!isAdmin` flag to the `disabled` var?

Reply:
This should work but moved to the disabled boolean